azure-pipelines-dependency-graph-testing.yml

```diff
@@ -97,11 +97,10 @@ stages:
             displayName: 'Use Python 3.7'
 
           - script: |
-              pip install isort[requirements]==5.10.1 flake8==3.8.3 black==22.3.0 pyupgrade==2.7.2
+              pip install isort[requirements]==5.10.1 flake8==3.9.2 black==22.3.0 pyupgrade==2.7.2 invoke==1.7.1
               EXIT_STATUS=0
-              isort . --check-only || EXIT_STATUS=$?
-              black --check . || EXIT_STATUS=$?
-              flake8 great_expectations/core || EXIT_STATUS=$?
+              invoke fmt --check || EXIT_STATUS=$?
+              invoke lint || EXIT_STATUS=$?
```

Comment:
I meant to switch these over back when I originally implemented them but I missed these other pipeline files.